tests/unit/test_openstack_builder.py

```diff
@@ -184,7 +184,7 @@ def test__create_security_group():
                 flavor="test-flavor",
                 network="test-network",
                 proxy="test-proxy",
-                upload_cloud_name="test-cloud-2",
+                upload_cloud_names=["test-cloud-2"],
```

Comment:
Should we also have a case with multiple?

Reply:
Added a case for multiple! Thank you for the review :)